src/tbots.py

```diff
@@ -44,16 +46,38 @@
 
     command = ["bazel", args.action, target]
 
-    # Handle visualizer argument
-    if args.action not in "build":
+    # Trigger a debug build
+    if args.debug_build:
+        command += ["-c", "dbg"]
+
+    # If its a binary, then run under gdb
+    if args.action in "run" and args.debug_build:
+        command += ["--run_under=gdb"]
+
+    # Don't cache test results
+    if args.action in "test":
         command += ["--cache_test_results=false"]
-    if args.enable_visualizer and args.action in "run":
-        command += ["--", "--enable_visualizer"]
-    if args.enable_visualizer and args.action in "test":
-        command += ['--test_arg="--enable_visualizer"']
+    if args.action in "run":
+        command += ["--"]
+
+    # Handle stop_ai_on_start
+    if args.stop_ai_on_start:
+        if args.action in "run":
+            command += ["--enable_visualizer"]
```

Comment:
Is this meant to be `--stop_ai_on_start`?

Reply:
good catch fixed!